app/src/apiClient.js

```diff
@@ -13,3 +15,16 @@ export const addTask = async (name) => {
   });
   return response.json();
 };
+
+export const testAuth = async () => {
+  //FIXME: get JWT (need to resolve React-related error)
```

Comment:
Do you still need these?

Reply:
@LeiaQuesada We can work on how to get the token to all you API client calls in a future session.